types/busboy/v0/busboy-tests.ts

```diff
@@ -0,0 +1,261 @@
+import Busboy = require('busboy');
```

Comment:
```suggestion
import busboy = require('busboy');
```
this is not how they name things:
https://github.com/mscdex/busboy/compare/v0.3.1...v1.0.0#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R101
so:
```js
const busboy = require('busboy');
...
const bb = busboy({...});

Reply:
Oh, sorry, I was mislead by versioning.
Please do not leave v0, it's 3 years old (in source). This package depends on Node, so it impacts Node CI tests on DT (each version adds a time for Node CI tests)

Reply:
I removed v0